managing-os/hvm.md

```diff
@@ -10,38 +10,30 @@ redirect_from:
 ---
 
 Creating and using HVM (fully virtualized) domains
-=============================================================
+==================================================
 
 What are HVM domains?
 ---------------------
 
 HVM domains (Hardware VM), in contrast to PV domains (Paravirtualized domains), allow one to create domains based on any OS for which one has an installation ISO.  For example, this allows one to have Windows-based VMs in Qubes.
 
-Interested readers might want to check
-[this article](https://blog.invisiblethings.org/2012/03/03/windows-support-coming-to-qubes.html)
-to learn why it took so long for Qubes OS to support HVM domains
-(Qubes 1 only supported Linux based PV domains). As of
-Qubes 4, every VM is PVH by default, except those with attached PCI devices which are HVM.
-[See here](https://blog.invisiblethings.org/2017/07/31/qubes-40-rc1.html) for a discussion
-of the switch to HVM from R3.2's PV, and [here](/news/2018/01/11/qsb-37/)
-for changing the default to PVH.
+Interested readers might want to check [this article](https://blog.invisiblethings.org/2012/03/03/windows-support-coming-to-qubes.html) to learn why it took so long for Qubes OS to support HVM domains (Qubes 1 only supported Linux based PV domains). As of Qubes 4, every VM is PVH by default, except those with attached PCI devices which are HVM. [See here](https://blog.invisiblethings.org/2017/07/31/qubes-40-rc1.html) for a discussion of the switch to HVM from R3.2's PV, and [here](/news/2018/01/11/qsb-37/) for changing the default to PVH.
 
 Creating an HVM domain
 ----------------------
 
-First, let's create a new HVM domain. Use the `--hvm` switch to `qvm-create`, or choose HVM type in the Qubes Manager VM creation dialog box:
+With a GUI: in Qubes Manager VM creation dialog box choose the "HVM" type.
```

Comment:
On R4 there is no "HVM" type in that dialog. Instead, one should choose "Standalone qube not based on a template". If "install system from device" is selected (which is by default), then `virt_mode` will be set to `hvm` automatically.